tests/sentry/tasks/integrations/github/test_open_pr_comment.py

```diff
@@ -900,9 +878,17 @@ def test_comment_workflow_missing_pr(self, mock_metrics, mock_pr_filenames):
             "github_open_pr_comment.error", tags={"type": "missing_pr"}
         )
 
-    @patch("sentry.tasks.integrations.github.open_pr_comment.get_pr_files")
     @patch("sentry.tasks.integrations.github.open_pr_comment.metrics")
-    def test_comment_workflow_missing_org(self, mock_metrics, mock_pr_filenames):
+    def test_comment_workflow_missing_org(
+        self,
+        mock_metrics,
+        _,
```

Comment:
hmm do you know why the patch on the test itself is inserted in that ordering of the function arguments?

Reply:
the patch is inserted at the top, presumably after all of the patches attached to the test class

Reply:
Oh I see, there's two metric mocks